src/renderer/mod.rs

```diff
@@ -30,7 +30,9 @@ pub use rendered_window::{LineFragment, RenderedWindow, WindowDrawCommand, Windo
 #[derive(SettingGroup, Clone)]
 pub struct RendererSettings {
     position_animation_length: f32,
+    window_animation: bool,
     scroll_animation_length: f32,
+    smooth_scrolling: bool,
```

Comment:
It seems like only the `cmd_line.rs` file deals with multigrid stuff. The rest adds `window_animation` and `smoth_scrolling`, but no corresponding commandline options or environment variables. Is this intentional?

Reply:
No this makes sense. When multigrid is disabled, the rendering system renders into a single window and doesn't send relevant scrolling events. So no cmd_line argument is needed.